src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,689 @@
+          socialLogo:
+            'https://www.yld.io/static/logo_animated-832020608244057f6a9d73e80994ac4a.gif'
+        }}
+      />
+      <CaseStudyHero caseStudy={caseStudy} />
+
+      <GreyBackground>
+        <Grid>
+          {/* DoctorLink is a healthcare technology company with a mission to simplify the route to health and wellbeing for patients globally... */}
+          {shouldRender(data1) && (
+            <BlockRow
+              mobile={addPaddings('4', '4')}
+              smallTablet={addPaddings('4', '4')}
+              tablet={addPaddings('4', '4')}
+              desktop={addPaddings('6', '6')}
+              widthPerc="75%"
+            >
+              <FullWidthBlock data={normalise(data1)} />
+            </BlockRow>
+          )}
+        </Grid>
+      </GreyBackground>
+
+      <Grid>
+        {/* The prologue */}
+        {shouldRender(data2) && (
+          <BlockRow
+            alignCenter
+            mobile={addPaddings('5', '5')}
+            smallTablet={addPaddings('7', '7')}
+            tablet={addPaddings('7', '7')}
+            desktop={addPaddings('7', '8')}
+          >
+            <TextAndImageBlock data={normalise(data2)} />
+          </BlockRow>
+        )}
+
+        {/* Youtube Video */}
+        {shouldRender(data3) && <VideoBlock data={normalise(data3)} />}
+
+        {/* Finding common ground */}
+        {shouldRender(data4) && (
+          <BlockRow
+            mobile={addPaddings('4', '4')}
+            smallTablet={addPaddings('5', '5')}
+            tablet={addPaddings('6', '6')}
+            desktop={addPaddings('7', '7')}
+          >
+            <TextColumnsBlock data={normalise(data4)} />
+          </BlockRow>
+        )}
+      </Grid>
+
+      <BlueBackground>
+        <Grid>
+          {/* Seamless */}
+          {shouldRender(data5) && (
+            <BlockRow
+              mobile={addPaddings('4', '4')}
+              smallTablet={addPaddings('5', '5')}
+              tablet={addPaddings('6', '6')}
+              desktop={addPaddings('6', '6')}
+            >
+              <TextColumnsBlock data={normalise(data5)} colorReverse />
+            </BlockRow>
+          )}
+        </Grid>
+      </BlueBackground>
+
+      <Grid>
+        {/* A new visual language */}
+        {shouldRender(data6) && (
+          <BlockRow
+            mobile={addPaddings('4', '4')}
+            smallTablet={addPaddings('5', '5')}
+            tablet={addPaddings('6', '6')}
+            desktop={addPaddings('7', '6')}
+          >
+            <TextColumnsBlock data={normalise(data6)} />
+          </BlockRow>
+        )}
+
+        {/* Image - Continue / back */}
+        {shouldRender(data7) && (
+          <BlockRow>
+            <FullWidthBlock data={normalise(data7)} />
+          </BlockRow>
+        )}
+
+        {/* Hi I am Noto sans */}
+        {shouldRender(data8) && (
+          <BlockRow
+            rowReverse
+            alignCenter
+            mobile={addPaddings('5', '5')}
```

Comment:
I would just write `{{bottom: 5, top: 5}}`, the `addPaddings` function isn't flexible in that you can't have top and no bottom and is another step that doesn't give us a huge advantage.

Reply:
Yeah. I also thought of that. I'm keen to go with it but we should discuss it too as I want to make sure we won't get into blockers in the future by doing that. I will investigate a bit now. We can discuss it after stand-up.